policy-build-rust/action.yaml

```diff
@@ -35,3 +35,35 @@ runs:
       shell: bash
       run: |
         kwctl annotate -m ${{ inputs.metadata-file }} -o annotated-policy.wasm ${{ inputs.input-wasm }}
+    -
+      name: Generate the SBOM files
+      shell: bash
+      run: |
+        curl -L -o $RUNNER_TEMP/spdx-sbom-generator-v0.0.15-linux-amd64.tar.gz https://github.com/opensbom-generator/spdx-sbom-generator/releases/download/v0.0.15/spdx-sbom-generator-v0.0.15-linux-amd64.tar.gz
+        tar -xf spdx-sbom-generator-v0.0.15-linux-amd64.tar.gz --directory $RUNNER_TEMP
```

Comment:
I would have a dedicated installer action, like we do with opa and other tools: https://github.com/kubewarden/github-actions/tree/v1/opa-installer

This improves reuse across different kind of policies (Go, Swift, Rust,...)

Reply:
Done.